### ISSUE TITLE
cuda section replacement

### ISSUE THREAD

**Issue by chensiyuan030105** [lzhnb/Analytic-Splatting#3]
Are the major changes to your code all in the cuda section? Can I directly replace the cuda part of 3DGS directly with the cuda part of your code to achieve anti-aliasing effect? I tried it and found that the psnr was very low, was there something I missed or misunderstood?

**Comment by lzhnb** [comment]
Yes, I mainly modify the CUDA code (`forward.cu` and `backward.cu`), and I also format parameters in function definition in CUDA.

**Comment by chensiyuan030105** [comment]
Thanks，however after I replace cuda section in original 3dgs by ana-gs，it seems that psnr is very low.


-----原始邮件-----
发件人: Zhihao ***@***.***>
目标语言: lzhnb ***@***.***>
抄送: Chen ***@***.***>; Author ***@***.***>
日期: 2024年7月25日星期四 09:43 CST
主题: Re: [lzhnb/Analytic-Splatting] cuda section replacement (Issue #3)


Yes, I mainly modify the CUDA code (`forward.cu` and `backward.cu`), and I also format parameters in function definition in CUDA.
